src/deployments.json

```diff
@@ -0,0 +1,31 @@
+{
+    "tokens": [
+        {
+            "name": "ETH",
+            "symbol": "ETH",
+            "address": "0x286f531a6406b26bb76dbb0c7f66fe793c184ab718b3e81a654855c3e32651ea",
+            "decimals": 18,
+            "minter": "contracts:dripper",
+            "upgrade_authority": "0x0eb1e6450cd74aaf5ef51ddbbad80beeb020c8aa79e18573ef383285b5dfb495"
```

Comment:
ideally we should add the deployed version with the contract class id (or a reference to the js released package), so that we can keep track of historical versions.
or, since this is a current state repo / file, we could have some test to check that building the repo and these addresses codebase matches